cypress/e2e/consent.cy.ts

```diff
@@ -71,6 +112,10 @@ describe.skip('tcfv2 consent', () => {
 				JSON.stringify({ value: 'GB' }),
 			);
 		});
+
+		Cypress.on('window:before:load', (win) => {
+			cy.spy(win.localStorage, 'setItem').log(false);
```

Comment:
I appreciate this is subjective but spying on DOM apis feels more like a unit/integration test territory rather than e2e. Is there a visual / DOM way this behaviour could be asserted instead?

Reply:
This is to check that the localStorage cookie reason is set, which is covered by a unit test, so we could leave it out for integration testing?

Reply:
If its covered by a unit test then yes I think it can be removed from the e2e test.

But happy to keep if you would prefer to have the safety of asserting here too.

Reply:
I think it's also handy to have cypress wait for `localstorage.setItem` to be called, instead of immediately tripping up if the value is not set quite yet.